framework/wazuh/core/tests/test_utils.py

```diff
@@ -1782,6 +1782,22 @@ def test_validate_wazuh_xml_ko(effect, expected_exception):
         with pytest.raises(WazuhException, match=f'.* {expected_exception} .*'):
             utils.validate_wazuh_xml(input_file)
 
+@pytest.mark.parametrize('effect, expected_exception', [
+    (utils.WazuhError, 1113)
+])
+def test_validate_wazuh_xml_ko_2(effect, expected_exception):
```

Comment:
Not sure what is the policy for multiple test cases for the same method, I used `test_validate_wazuh_xml_ko_2` since I couldn't find a reference